source/migrating_api.md

```diff
@@ -0,0 +1,450 @@
+
+---
+
+
+### Form endpoints
+These endpoints return detailed attributes of all forms shared with you or about a specific form, where `{uid}` is the unique identifier of the project. 
+
+**Endpoint mapping**
+
+| `v1` Endpoint    | `v2` Equivalent        |
+| ------------- | ---------------------- |
+| `/api/v1/forms`   | `/api/v2/assets/?asset_type=survey` |
+| `/api/v1/forms/<pk>`       | `/api/v2/assets/{asset_uid}/`                 |
+
+
+<p class="note">
+  <b>Note:</b> The <code>v2</code> endpoint follows the same structure for each item as listed below, but introduces pagination. Some properties from the <code>v1</code> endpoint are not directly available on the <code>v2</code> asset endpoint, but they can still be accessed differently (see the legend below the table for more details).
+</p>
+
+
+**Field mapping**
+| `v1` Field                 | `v2` Equivalent                          |
+|----------------------------|------------------------------------------|
+| `formid`                   | `uid`<sup>1</sup>                        |
+| `owner`                    | `owner__username`                        |
+| `metadata`                 | `files`                                  |
+| `tags`                     | `tag_string`<sup>2</sup>                 |
+| `title`                    | `name`                                   |
+| `public`                   | _N/A_<sup>3</sup>                        |
+| `public_data`              | _N/A_<sup>3</sup>                        |
+| `require_auth`             | _N/A_<sup>3</sup>                        |
+| `users`                    | `permissions`                            |
+| `hash`                     | `version__content_hash`                  |
+| `downloadable`             | `deployment__active`                     |
+| `encrypted`                | `deployment__encrypted`                  |
+| `last_submission_time`     | `deployment__last_submission_time`       |
+| `uuid`                     | `deployment__uuid`                       |
+| `instances_with_geopoints` | `summary.geo`                            |
+| `num_of_submissions`       | `deployment__submission_count`           |
+| `attachment_storage_bytes` | _N/A_<sup>4</sup>                        |
+
+<sup>1</sup> _In the `/api/v2/assets` endpoint, sequential integer identifiers are no longer used. Each entry is uniquely identified by an alphanumeric `uid`_.  
+<sup>2</sup> _In `v1`, tags were returned as an array; in `v2`, they are returned as a comma-separated string._  
+<sup>3</sup> _These fields are no longer exposed. See the **Permissions** section below for more details._  
+<sup>4</sup> _Not directly accessible via the asset endpoint. Use the `/api/v2/asset_usage/` endpoint and retrieve the `storage_bytes` field of the corresponding project._
+
+<details>
+<summary><strong>Example <code>v1</code> response</strong></summary>
+<br>
+  
+```json
+{
+  "url": "https://kf.kobotoolbox.org/api/v1/forms/474.json",
```

Comment:
My bad, when I provided the examples, I forgot to change the format so it should be: 

```
 "url": "https://kf.kobotoolbox.org/api/v1/forms/474",
 ```